controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller_intg_test.go

```diff
@@ -33,32 +33,24 @@ func cclItemReconcile() {
 	waitForClusterVirtualMachineImageReady := func(ctx *builder.IntegrationTestContext) {
 		curCVMI := vmopv1.ClusterVirtualMachineImage{}
 		expectedCVMI := utils.GetExpectedCVMIFrom(*cclItem, intgFakeVMProvider.SyncVirtualMachineImageFn)
-		Eventually(func() bool {
+		Eventually(func(g Gomega) error {
 			cvmiName := utils.GetTestVMINameFrom(cclItem.Name)
 			if err := ctx.Client.Get(ctx, client.ObjectKey{Name: cvmiName}, &curCVMI); err != nil {
```

Comment:
You can simplify this more by `g.Expect(err).ToNot(HaveOccurred())` here, and then change the func's return type to unit/void. Gomega will treat a non-assertion function as succeeding then.